### PR TITLE
Expand experts

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8649/

File: jujugui/static/gui/src/app/components/expert-card/experts.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 module.exports = {
-  spicule: {
+  spiculecharms: {
```
Comment (mitechie) [inline jujugui/static/gui/src/app/components/expert-card/experts.js]: is there any code that needs to pick up this change?

File: jujugui/static/gui/src/app/components/expert-card/experts.js [hunk 2]
```diff
@@ -21,5 +21,22 @@ module.exports = {
     storeDescription: (
       'Learn how Spicule and Canonical can help solve your Big Data ' +
       'challenges with JAAS:')
+  },
+  'tengu-team': {
+    logo: 'tengu.png',
+    highlights: [
+      'Your own Big Data workspace',
+      'Use all Juju-charmed technologies',
```
Comment (mitechie) [inline jujugui/static/gui/src/app/components/expert-card/experts.js]: is this supposed to be "Uses"?

File: jujugui/static/gui/src/app/components/expert-card/experts.js (hunk 1 above)
Comment (hatched) [reply jujugui/static/gui/src/app/components/expert-card/experts.js]: Nope, its just a map that gets checked when an entity is deployed.

File: jujugui/static/gui/src/app/components/expert-card/experts.js (hunk 2 above)
Comment (hatched) [reply jujugui/static/gui/src/app/components/expert-card/experts.js]: This was taken from the copy that was supplied, I imagine it'll get an overhaul once it's on staging...

**Comment by hatched** [comment]
Thanks @mitechie :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8650/

**Pull request merged**